test/Microsoft.Azure.Devices.Edge.Test.Common/IOsPlatform.cs

```diff
@@ -19,16 +19,16 @@ public interface IOsPlatform
         // After calling this function, the following files will be available under {scriptPath}:
         //  certs/iot-device-{deviceId}-full-chain.cert.pem
         //  private/iot-device-{deviceId}.key.pem
-        Task<Certificates> CreateDeviceCertificatesAsync(string deviceId, string scriptPath, CancellationToken token);
+        Task<IdCertificates> GenerateIdentiyCertificatesAsync(string deviceId, string scriptPath, CancellationToken token);
```

Comment:
Typo in method name: Identiy

Reply:
`GenerateIdentiyCertificatesAsync()` -> `GenerateIdentityCertificatesAsync()`